src/Servant/Server/Experimental/Auth/Cookie.hs

```diff
@@ -441,6 +442,28 @@ addSession acs rs sk sessionData response = do
   header <- renderSession acs rs sk sessionData
   return (addHeader (EncryptedSession header) response)
 
+-- |  "Remove" a session by invalidating the cookie.
+-- Cookie expiry date is set at 0  and content is wiped
+removeSession  :: ( MonadIO m
+                  , MonadThrow m
+                  , AddHeader (e :: Symbol) ByteString s r )
+  => AuthCookieSettings -- ^ Options, see 'AuthCookieSettings'
+  -> s                 -- ^ Response to return with  session removed
+  -> m r               -- ^ Response with the session "removed"
+removeSession acs@AuthCookieSettings{..} response = 
```

Comment:
We don't use this argument in the body, so it would be better to prefix it with underscore, like `_acs`.